ProtocolHandler.pm

```diff
@@ -1,87 +1,112 @@
 package Plugins::RadioParadise::ProtocolHandler;
 
 use strict;
-# XXX - https
-use base qw(Slim::Player::Protocols::HTTP);
+use base qw(IO::Handle);
 
 use JSON::XS::VersionOneAndTwo;
 use Tie::Cache::LRU;
 use POSIX qw(ceil);
-
-use Slim::Networking::SimpleAsyncHTTP;
+use List::Util qw(min max);
+use AnyEvent::DNS;
```

Comment:
You're not really using this, are you?

Reply:
no I don't anywore. I used it at the beginning and it was way too slow

Reply:
Can you remove it then?